microprofile-jwt/pom.xml

```diff
@@ -83,5 +83,129 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>bootablejar.profile</id>
+            <properties>
+                <!-- TBD - remove once https://issues.redhat.com/browse/WFARQ-74 is done and wildfly-arquillian-3.0.0.Beta2 is out -->
+                <jboss.home>${project.build.directory}</jboss.home>
+            </properties>
+            <activation>
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This and the following plugins config to disabled them is handled at the parent POM level and can be removed.